Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -270,8 +327,8 @@ namespace gdjs {
       //No acceleration for angular speed for now
 
       //Position object
-      object.setX(object.getX() + this._xVelocity * timeDelta);
-      object.setY(object.getY() + this._yVelocity * timeDelta);
+      this._x += this._xVelocity * timeDelta;
+      this._y += this._yVelocity * timeDelta;
```

Comment:
Do you think that instead of converting from "screen" coordinates to "world" coordinates and back to "screen" coordinates, we could instead only multiply the changes done on the x and y position by some factor, depending on the viewpoint chosen (x: 1, y: 1factors in the case of top down,  x: 1, y: 0.5 in the case of pixel isometry)?

Would that impact other things like acceleration or movement? 
I'm not a specialist in how to deal with isometry, but I'm wondering if it would be sufficient (and more performant than doing point space transformations at every frame).

Reply:
Indeed, I used the "in world" position for the other pull request but that's not necessary here. Only the delta can be transformed.

About the pixel isometry, to keep the distance there is a factor:
screenX = 0.5*sqrt(2) * (worldX - worldY)
screenY = 0.5 * 0.5*sqrt(2) * (worldX + worldY)

Reply:
> So toWorld() becomes a dead code, should I remove it?

Yes please! I know it's a bit frustrating but in this context I would really like to have the minimum changes required to make this work.
The other Pull Request is pretty big, and maybe the behavior will need rework/be separated in another behavior - but for this PR let's keep things dead simple.

> I think what you said should work but the up will still be up by default and the transformation is just 6 operations that's not really costly. What do you think?

Well it's only 6 operations but it's still 6 more operations even in the basic case, so if someone uses this behavior for animating 100 small objects on screen (to benefit from acceleration, rotation, etc...), it's now 600 operations.

Might not be too much (depends on the device), but still I kind of like the idea that if you keep the top down approach, we don't add any overhead to the behavior every frame (if you're using isometric coordinates, that's ok) :)

> but the up will still be up by default

I think that's fine, it would be easy to move from topdown to isometric, and creators can still remap the controls if they need using events?

Reply:
> I think that's fine, it would be easy to move from topdown to isometric, and creators can still remap the controls if they need using events?
Yes, they can do it with the angle offset property. But, imho, even though it's hard for the brain,  it's more usable to go up in diagonal because the world is likely built to go that way. Imagine a long corridor or a labyrinth where you have to keep pressing 2 keys not to take a wall.

Reply:
Sorry I missed your reply!
I'll take a new look at the code :)